packages/common/src/JsonRpcClient.ts

```diff
@@ -86,6 +90,16 @@ export class JsonRpcClient {
         return res.result;
     }
 
+    /**
+     * @param serializedTransaction the transaction serialized as a base64-encoded string.
+     */
+    async sendRawTransaction(serializedTransaction: string): Promise<boolean> {
+        const res = await this.provider.request('sendAccountTransaction', {
```

Comment:
We should update the name from `sendAccountTransaction` to `sendTransaction` if the JSON-RPC endpoint supports all transaction types anyway.

Reply:
https://github.com/Concordium/concordium-json-rpc/pull/16